### PR TITLE
[one-cmds] Add requantize option to one-quantize

### PR DESCRIPTION
**Pull request by jinevening**
This adds requantize option to one-quantize.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/11109
Draft PR: https://github.com/Samsung/ONE/pull/11155

### CODE REVIEW

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -258,6 +267,21 @@ def _set_default_values(args):
...
 
+def _verify_arg_pre(parser, args):
+    """verify given arguments before default values are set"""
+    # check if required arguments is given
+    missing = []
+    if oneutils.is_valid_attr(args, 'requantize'):
+        if not oneutils.is_valid_attr(args,
+                                      'input_model_dtype') and not oneutils.is_valid_attr(
+                                          args, 'input_dtype'):
+            missing.append('--input_model_dtype')
```
Comment (seanshpark) [inline compiler/one-cmds/one-quantize:278]: Q) what is the relation with `input_dtype` and `--input_model_dtype` ?

Comment (seanshpark) [reply compiler/one-cmds/one-quantize:278]: Um... to use `--requantize` what are the required inputs(options?) and optional inputs(options) ?

Comment (jinevening) [reply compiler/one-cmds/one-quantize:278]: `input_dtype` was introduced first to designate the dtype of input model, but it is deprecated (in normal quantize option). Now `input_model_dtype` is used instead.

So, `input_model_dtype` is float32 for normal quantization, and int8 for requantization.

> Um... to use --requantize what are the required inputs(options?) and optional inputs(options) ?

Required inputs are
1. `input_model_dtype` (or `input_dtype` can be used for backward compatibility) -> int8
2. `quantized_dtype` -> uint8
3. `input_path` -> int8 circle
4. `output_path` -> uint8 circle

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**